scripts/Addicted/System.reds

```diff
@@ -192,24 +192,25 @@ public class AddictedSystem extends ScriptableSystem {
       E(s"no consumption tracked during prologue");
       return;
     }
-    let before: Threshold;
-    let after: Threshold;
-    let amount: Int32;
+    let previousThreshold: Threshold;
+    let newConsumption: Int32;
     let hint: Bool;
-    if Generic.IsAddictive(id) {
```

Comment:
could you please avoiding renaming the vars when it's not necessary, it avoids "noise" in the review :)

Reply:
reverted :D